src/analyzing/tests/Test_analyze_generic.ml

```diff
@@ -89,8 +88,10 @@ let test_dfg_svalue ~parse_program file =
       inherit [_] AST_generic.iter_no_id_info
 
       method! visit_function_definition _ def =
-        let inputs, xs = AST_to_IL.function_definition lang def in
-        let flow = CFG_build.cfg_of_stmts xs in
+        let inputs, flow =
+          match CFG_build.cfg_of_fdef lang def with
+          | { fparams; fcfg } -> (fparams, fcfg)
```

Comment:
You can write this as `let {fparams = inputs; fcfg = flow} = CFG_build.cfg_of_fdef lang def in`.

Reply:
Done.